build/pyodide_bucket.bzl

```diff
@@ -1,8 +1,178 @@
 # Do not edit this file by hand. See docs/pyodide.md for info on how to generate it.
 # These variables are factored out here because they are being shared by the WORKSPACE files in
 # both edgeworker and workerd, as well as src/pyodide/BUILD.bazel
-PYODIDE_PACKAGE_BUCKET_URL = "https://pub-45d734c4145d4285b343833ee450ef38.r2.dev/20240415-experimental/"
-PYODIDE_GITHUB_RELEASE_URL = "https://github.com/cloudflare/pyodide-build-scripts/releases/download/20240415-experimental/"
-PYODIDE_LOCK_SHA256 = "67d1a24edf4f3ab2cf85c736391c04763ff722bf3aebf9ea3469d96e5f51e1da"
-PYODIDE_PACKAGES_TAR_ZIP_SHA256 = "749967941204154e7ae866fe08f1216a3e5ee58ba6a3757231a5be0d9d4430f8"
-PYODIDE_ALL_WHEELS_ZIP_SHA256 = "9e7c330ee93d81d0356cc2d585f217dfee58b623ad4535282baa6e82bd063eee"
+PYODIDE_PACKAGE_BUCKET_URL = "https://pub-45d734c4145d4285b343833ee450ef38.r2.dev/20240508/"
+PYODIDE_GITHUB_RELEASE_URL = "https://github.com/cloudflare/pyodide-build-scripts/releases/download/20240508/"
+PYODIDE_LOCK_SHA256 = "697b8917a8dcedbb78dd21fa1927ee310604106579e93b1eac425344d9212a73"
+PYODIDE_PACKAGES_TAR_ZIP_SHA256 = "12238b283ab2c2b996fcec73aa9e61f057c86aad5d612c2d44f4b2764f7ea03a"
+PYODIDE_ALL_WHEELS_ZIP_SHA256 = "36f980ae303357410bc4a34b430bcf82631e80525ac144888c696bcf92d03504"
+
+# IMPORTANT: when updating this file in git, check the diff to make sure none of the imports below are being removed unexpectedly
+PYODIDE_IMPORTS_TO_TEST = {
```

Comment:
Do we have a script that can regenerate this? I think it would be useful and it could also verify that no existing packages are being removed.

Reply:
It's generated by `pyodide-build-scripts`. This whole file is part of the release that's generated by the GitHub action.

Reply:
For reference here is the script: https://github.com/cloudflare/pyodide-build-scripts/pull/2